src/Main.java

```diff
@@ -1,5 +1,79 @@
+import java.text.NumberFormat;
+import java.util.Random;
+
 public class Main {
+    public static NumberFormat numberFormat = NumberFormat.getCurrencyInstance();
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        task1();
+        task2();
+        task3();
+        task4();
     }
-}
\ No newline at end of file
+
+    public static int[] generateRandomArray() {
+
+        Random random = new Random();
+        int[] arr = new int[30];
+
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+        return arr;
+    }
+
+    public static void task1() {
+
+        int[] monthlyPayments = generateRandomArray();
+        int sumOfAllPayments = 0;
+
+        for (int monthlyPayment : monthlyPayments) {
+            sumOfAllPayments += monthlyPayment;
+        }
+        System.out.printf("Сумма всех выплат за месяц составила %s руб.\n",
+                numberFormat.format(sumOfAllPayments));
+    }
+
+    public static void task2() {
+
+        int[] monthlyExpenses = generateRandomArray();
+        int minExpenses = Integer.MAX_VALUE;
+        int maxExpenses = 0;
+
+        for (int monthlyExpense : monthlyExpenses) {
+
+            if (monthlyExpense > maxExpenses) {
+                maxExpenses = monthlyExpense;
+            } else if (monthlyExpense < minExpenses) {
```

Comment:
```suggestion
            }
            if (monthlyExpense < minExpenses) {
```
потому что в массиве из 1 элемента у тебя найдёт только максимум, а минимум будет равен `Integer.MAX_VALUE`, что неверно